apps/admin/src/app/_components/challenge-reviews/index.tsx

```diff
@@ -8,11 +8,9 @@ import { columns } from './columns';
 export function ChallengeReviews() {
   const { data, isLoading } = useQuery(['challenge-reviews'], () => getPendingChallenges());
 
-  if (isLoading || !data) return null;
-
   return (
     <div className="w-full rounded-md border">
-      <DataTable data={data} columns={columns} />
+      {data ? <DataTable data={data} columns={columns} /> : ''}
```

Comment:
Similar to the other change, you're just displacing where the empty check is. This doesn't solve the supposed issue.